app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <string name="logs_view_button">Zobrazit logy…</string>
+  <string name="logs_view_with">Zobrazit logy pomocí…</string>
+  <string name="logs_share_button">Sdílet logy…</string>
+  <string name="logs_share_subject">Phoenix App logy</string>
+  <string name="logs_share_title">Sdílet Phoenix logy…</string>
+
+  <!-- //////////////// payment details //////////////// -->
+
+  <string name="paymentdetails_loading">Získávání detailů platby…</string>
+  <string name="paymentdetails_error">Nastala chyba při získávání detailů platby.\n\nProsím, zkuste to znovu.</string>
+
+  <string name="paymentdetails_status_sent_successful"><![CDATA[<b>Odesláno</b> %1$s]]></string>
+  <string name="paymentdetails_status_sent_pending"><![CDATA[<b>Čekám…</b>]]></string>
+  <string name="paymentdetails_status_sent_failed"><![CDATA[Platba <b>selhala</b>.<br />Žádné peníze nebyly odeslány.]]></string>
+
+  <string name="paymentdetails_status_received_pending">Platba zatím nebyla obdržena.</string>
+  <string name="paymentdetails_status_received_successful"><![CDATA[<b>Obdrženo</b> %1$s]]></string>
+
+  <string name="paymentdetails_desc_label">Popis</string>
+  <string name="paymentdetails_destination_label">Odesláno k</string>
+  <string name="paymentdetails_fees_label">Poplatky</string>
+  <string name="paymentdetails_error_label">Chyba</string>
+  <string name="paymentdetails_closing_desc">Platba k uzavření kanálu #%1$s</string>
+  <string name="paymentdetails_no_description">Žádný popis</string>
+
+  <string name="paymentdetails_technicals_button">Zobrazit technická data</string>
+
+  <!-- //////////////// payment details: technicals data //////////////// -->
+
+  <string name="paymentdetails_technicals_title">Platební data</string>
+  <string name="paymentdetails_payment_hash_label">Platební Hash</string>
+  <string name="paymentdetails_payment_request_label">Platební Požadavek</string>
+  <string name="paymentdetails_preimage_label">Preimage</string>
+  <string name="paymentdetails_created_at_label">Vytvořeno v</string>
+  <string name="paymentdetails_expire_at_label">Vyprší v</string>
+  <string name="paymentdetails_completed_at_label">Dokončeno v</string>
+
+  <!-- //////////////// payment details: technicals data //////////////// -->
+
+  <string name="state_wait_confirmed">Spojovací transakce čeká na potvrzení</string>
+  <string name="state_closing">Kanál se uzavírá</string>
+  <string name="state_closed">Kanál byl uzavřen</string>
+  <string name="state_normal">Běží v pořádku</string>
+  <string name="state_sync">Synchronizuji s uzlem</string>
+  <string name="state_offline">Uzel je offline</string>
+  <string name="state_shutdown">Vypnuto</string>
+
+  <!-- //////////////// seed security //////////////// -->
+
+  <string name="seedsec_title">Bezpečnost</string>
+  <string name="seedsec_subtitle">Vysoce doporučujeme nastavit PIN k ochraně vaší peněženky. Pokud to vaše zařízení podporuje, můžete použít biometrické přihlášení.</string>
```

Comment:
Pokud jej vaše zařízení...

Reply:
That would be misleading IMO. Sounds like "if the wallet supports PIN, you can use biometrics..."

Reply:
True. We would have to change the order to: "Vysoce doporučujeme nastavit PIN k ochraně vaší peněženky. Můžete použít biometrické přihlášení, pokud jej vaše zařízení podporuje."  That's a bit overkill.

Reply:
Right.